cpp/demo/interpolation/main.cpp

```diff
@@ -0,0 +1,73 @@
+#include <algorithm>
+#include <cmath>
+
+#include <dolfinx.h>
+#include <dolfinx/io/XDMFFile.h>
+
+#include "interpolation.h"
+
+using namespace dolfinx;
+
+int main(int argc, char *argv[])
+{
+  common::subsystem::init_logging(argc, argv);
+  common::subsystem::init_petsc(argc, argv);
+
+  // Set the logging thread name to show the process rank
+  int mpi_rank;
+  MPI_Comm_rank(MPI_COMM_WORLD, &mpi_rank);
+  std::string thread_name = "RANK " + std::to_string(mpi_rank);
+  loguru::set_thread_name(thread_name.c_str());
+  if (dolfinx::MPI::rank(MPI_COMM_WORLD) == 0) {
+      loguru::g_stderr_verbosity = loguru::Verbosity_INFO;
+  }
+
+  {
+    auto meshL = std::make_shared<mesh::Mesh>(generation::BoxMesh::create(
+        MPI_COMM_WORLD, {{{0.0, 0.0, 0.0}, {1.0, 1.0, 1.0}}}, {16, 16, 16},
+        mesh::CellType::tetrahedron, mesh::GhostMode::shared_facet));
+
+    auto meshR = std::make_shared<mesh::Mesh>(generation::BoxMesh::create(
+        MPI_COMM_WORLD, {{{0.25, 0.25, 0.25}, {0.75, 0.75, 0.75}}},
+        {16, 16, 16}, mesh::CellType::tetrahedron,
+        mesh::GhostMode::shared_facet));
```

Comment:
What is the interpolation behaviour if you would do `interpolate(uL, uR)` where several points will find no candidate on the other mesh?

Reply:
If a point in the target mesh cannot be evaluated by the source mesh, its value is set to zero.